app/index.js

```diff
@@ -131,10 +131,9 @@ const useragent = () => session.defaultSession.getUserAgent()
     .replace(RegExp(`${app.getName()}/\\S* `), "")
 
 // Parse arguments
-let args = process.argv.slice(1)
-if (args[0] === "app" || args[0] === app.getAppPath()) {
-    args = args.slice(1)
-}
+// If process.defaultApp is true, then argv is ["electron", "app", ...args].
+// Otherwise, argv is ["vieb", ...args].
```

Comment:
Please refactor this code to work on regular systems, `process.defaultApp` is undefined there, which means that this calculation evaluates to `undefined + 1` which is actually `NaN` (instead of 1, like you probably assumed). This refactor also gives opportunity to better use variables to hopefully require less explicit comments (though please do add them back if they are required for understanding it).

Reply:
Oh wow, this was not a boolean?!?  I've changed it and tested it in an ubuntu vm.

Reply:
It depends, it's true when it's there, but otherwise it's undefined.